doc/THIRD-PARTY-LICENSES

```diff
@@ -54,44 +48,44 @@ License: public-domain
  You can use this free for any purpose.  It's in the public domain.
  It has no warranty.
 
-Files: src/tools/rest-frontend/resources/assets/skin/base/util/space.less
+Files: src/tools/website/resources/assets/skin/base/util/space.less
 Copyright: 2015 Ömer Aslanbakan
 License: MIT
 
-Files: src/tools/rest-frontend/public/assets/fonts/lato/*
+Files: src/tools/website/public/assets/fonts/lato/*
 Copyright: 2010-2014 by tyPoland Lukasz Dziedzic (team@latofonts.com) with Reserved Font Name "Lato"
 License: OFL-1.1
 
-Files: src/tools/rest-frontend/public/assets/fonts/libre-franklin/*
+Files: src/tools/website/public/assets/fonts/libre-franklin/*
 Copyright: 2015 Impallari Type (www.impallari.com)
 License: OFL-1.1
 
-Files: src/tools/rest-frontend/public/assets/fonts/font-awesome/*
+Files: src/tools/website/public/assets/fonts/font-awesome/*
 Copyright: 2016 Dave Gandy <dave@fontawesome.io>
 License: OFL-1.1
 
-Files: src/tools/rest-frontend/public/vendor/asciinema/asciinema-player.*
+Files: src/tools/website/public/vendor/asciinema/asciinema-player.*
 Copyright: 2011-2016 Marcin Kulik
 License: Apache-2.0
 
-Files: src/tools/rest-frontend/resources/assets/skin/base/fonts/font-awesome/*
+Files: src/tools/website/resources/assets/skin/base/fonts/font-awesome/*
 Copyright: 2016 Dave Gandy <dave@fontawesome.io>
 License: MIT
 
 Files: src/tools/qt-gui/src/markdownconverter/*
 Copyright: 2013 Christian Loose
 License: BSD-3-clause-markdownconverter
 
-Files: scripts/realpath
+Files: scripts/dev/realpath
```

Comment:
I added further changes that were not explicitly asked for where I found invalid paths